identity/identity.go

```diff
@@ -13,19 +13,22 @@ import (
 type Author interface {
 	AuthorID() string
 	AuthorPubKey() crypto.PubKey
+	AuthorName() string
```

Comment:
should be `Username() string`.  This entire `author` package should get refactored away in the near future, ideally removing the `Author` terminology entirely.